tests/integration/utils.sh

```diff
@@ -38,6 +38,38 @@ function build_distro_image()
     fail "(${LINENO}): Error building the image for distribution ${distro}"
     return "$?"
   fi
+
+  local size_unit
+  local size
+  local id
+
+  size_unit=$(podman image inspect --format '{{ .Size }}' "kw-${distro}")
+  id=$(podman image inspect --format '{{ .Id }}' "kw-${distro}")
+
+  # Converting size in bytes to the appropriate units
+
+  if [ "$size_unit" -lt 1024 ]; then
+    size="${size_unit} B"
+
+  elif [ "$size_unit" -lt "$(bc <<< 1024^2)" ]; then
+    size_unit=$(echo "scale=2; $size_unit / 1024" | bc)
+    size="${size_unit} KB"
+
+  elif [ "$size_unit" -lt "$(bc <<< 1024^3)" ]; then
+    mb_size=$(bc <<< 1024^2)
+    size_unit=$(echo "scale=2; $size_unit / ${mb_size}" | bc)
+    size="${size_unit} MB"
+
+  else
+    gb_size=$(bc <<< 1024^3)
+    size_unit=$(echo "scale=2; $size_unit / ${gb_size}" | bc)
+    size="${size_unit} GB"
+
+  fi
+
+  echo "Cached container distribution: ${distro}; Image ID: ${id}; Repository: ${file}; Size: ${size}"
```

Comment:
The current output format is a bit hard to read:

![image](https://github.com/kworkflow/kworkflow/assets/43859386/6678be39-9752-4614-89fe-9a54a40bae2c)

To improve readability, consider using the warning function for each piece of information:

Something like:

```
warning "Cached container distribution: ${distro}"
warning "Image ID: ${id}"
```

This will produce a cleaner, more readable output :)